src/Views/AlertList/Types.elm

```diff
@@ -0,0 +1,14 @@
+module Views.AlertList.Types exposing (AlertListMsg(..), Route(..))
+
+import Utils.Types exposing (ApiData, Filter)
+import Alerts.Types exposing (Alert, AlertGroup)
+
+
+type Route
+    = Receiver (Maybe String) (Maybe Bool) (Maybe String)
```

Comment:
Definitely something to note -- I like how this is self documenting, while in what I originally wrote it's a total mystery what the different `Maybe`s represent.